app/models/task.py

```diff
@@ -1,5 +1,38 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import String, DateTime, ForeignKey
+from datetime import datetime
+from typing import Optional
 from ..db import db
 
 class Task(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str] = mapped_column(String(50))
```

Comment:
As with `Goal`, consider which of the columns for `Task` should be nullable and which should not be.